Comanche055/INTER-BANK_COMMUNICATION.agc

```diff
@@ -145,27 +145,27 @@ USPRCADR	TS	LOC		# SAVE A
 # THERE ARE FOUR POSSIBLE SETTINGS FOR CHANNEL 07.  (CHANNEL 07 CONTAINS SUPERBANK SETTING.)
 #
 #					PSEUDO-FIXED	 OCTAL PSEUDO
-# SUPERBANK	SETTING	S-REG. VALUE	BANK NUMBERS	   ADDRESSES
-# ---------	-------	------------	------------ 	   ---------
-# SUPERBANK 3	  OXX	 2000 - 3777	   30 - 37	 70000 - 107777		(WHERE XX CAN BE ANYTHING AND
+# SUPERBANK	SETTING	S-REG. VALUE	BANK NUMBERS	 ADDRESSES
+# ----------	-------	------------	------------ 	  ------------
+# SUPERBANK 3	  0XX	2000 - 3777	   30 - 37	 70000 - 107777		(WHERE XX CAN BE ANYTHING AND
 #										WILL USUALLY BE SEEN AS 11)
-# SUPERBANK 4	  100	 2000 - 3777	   40 - 47	110000 - 127777		(AS FAR AS IT CAN BE SEEN,
+# SUPERBANK 4	  100	2000 - 3777	   40 - 47	 110000 - 127777	(AS FAR AS IT CAN BE SEEN,
 #										ONLY BANKS 40-43 WILL EVER BE
 #										AND ARE PRESENTLY AVAILABLE)
-# SUPERBANK 5	  101	 2000 - 3777	   50 - 57	130000 - 147777		(PRESENTLY NOT AVAILABLE TO
+# SUPERBANK 5	  101	2000 - 3777	   50 - 57	 130000 - 147777	(PRESENTLY NOT AVAILABLE TO
 #										THE USER)
-# SUPERBANK 6	  110	 2000 - 3777	   60 - 67	150000 - 167777		(PRESENTLY NOT AVAILABLE TO
+# SUPERBANK 6	  110	2000 - 3777	   60 - 67	 150000 - 167777	(PRESENTLY NOT AVAILABLE TO
 #										THE USER)
-# *** THIS ROUTINE MAY BE CALLED BY ANY PROGRAM LOCATED IN BANKS 00 - 27.  I.E., NO PROGRAM LIVING IN ANY
+# *** THIS ROUTINE MAYBE CALLED BY ANY PROGRAM LOCATED IN BANKS 00 - 27.  I.E., NO PROGRAM LIVING IN ANY
 # SUPERBANK SHOULD USE SUPERSW. ***
 #
-# SUPERSW MAY BE CALLED IN THIS FASHION:
-#	CAF	ABBCON		WHERE -- ABBCON  BBCON  SOMETHING --
+# SUPERSW MAYBE CALLED IN THIS FASHION:
+#	CAF	ABBCON		WHERE -- ABBCON    BBCON   SOMETHIN --
```

Comment:
✔️ `SOMETHIN`

❌ Only 3 spaces between `ABBCON` and `BBCON`

❌ Only 2 spaces between `BBCON` and `SOMETHIN`

✨ Can you add double spaces after `WHERE`, `--` and `SOMETHIN`